tests/test_band_masks.py

```diff
@@ -44,28 +44,28 @@ def tiffs(tmpdir):
 
 def test_mask_flags():
     with rasterio.open('tests/data/RGB.byte.tif') as src:
-        for flags in src.mask_flags:
-            assert flags & MaskFlags.nodata
-            assert not flags & MaskFlags.per_dataset
-            assert not flags & MaskFlags.alpha
```

Comment:
Since `mask_flags` is still part of the public API, is there any value in keeping these direct tests? We still us the bitmask-style comparisons in `_io.pyx`.